external/netstandard/netstandard1.x/project.json.template

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "frameworks": {
+    "{TFM}": {
+      "dependencies": {
+        "NETStandard.Library": "2.0.0-beta-24919-01",
+        "System.Diagnostics.Contracts": "4.3.0"
```

Comment:
We can probably get away with sharing the same template and just having a poor mans version of your conflict resolution in the project.